salt/cert/init.sls

```diff
@@ -38,14 +38,14 @@ include:
     - signing_policy: minion
     - public_key: /etc/pki/minion.key
     - CN: {{ grains['fqdn'] }}
-    - C: {{ pillar['certificate_information']['subject_properties']['C'] }}
-    - Email: {{ pillar['certificate_information']['subject_properties']['Email'] }}
-    - GN: {{ pillar['certificate_information']['subject_properties']['GN'] }}
-    - L: {{ pillar['certificate_information']['subject_properties']['L'] }}
-    - O: {{ pillar['certificate_information']['subject_properties']['O'] }}
-    - OU: {{ pillar['certificate_information']['subject_properties']['OU'] }}
-    - SN: {{ pillar['certificate_information']['subject_properties']['SN'] }}
-    - ST: {{ pillar['certificate_information']['subject_properties']['ST'] }}
+    - C: "{{ pillar['certificate_information']['subject_properties']['C'] }}"
+    - Email: "{{ pillar['certificate_information']['subject_properties']['Email'] }}"
+    - GN: "{{ pillar['certificate_information']['subject_properties']['GN'] }}"
+    - L: "{{ pillar['certificate_information']['subject_properties']['L'] }}"
+    - O: "{{ pillar['certificate_information']['subject_properties']['O'] }}"
+    - OU: "{{ pillar['certificate_information']['subject_properties']['OU'] }}"
+    - SN: "{{ pillar['certificate_information']['subject_properties']['SN'] }}"
+    - ST: "{{ pillar['certificate_information']['subject_properties']['ST'] }}"
```

Comment:
We aren't embedding jinja into jinja... it's the fact that `NO` gets interpreted by the yaml parser as `False`, so quoting is the optimal solution IMHO.